AFQ/dki.py

```diff
@@ -71,7 +71,7 @@ def fit_dki(data_files, bval_files, bvec_files, mask=None,
     names = ['FA', 'MD', 'AD', 'RD', 'MK', 'AK', 'RK', 'params']
 
     if out_dir is None:
-        out_dir = op.join(op.split(data_files[0])[0], 'dki')
+        out_dir = op.join(op.split(data_files)[0], 'dki')
```

Comment:
Maybe it doesn't but I am not testing it enough... Let me see...

On Sun, May 8, 2016 at 8:20 PM, Ben Cipollini <notifications@github.com>
wrote:

> In AFQ/dki.py
> <https://github.com/yeatmanlab/pyAFQ/pull/21#discussion_r62447142>:
>
> > @@ -71,7 +71,7 @@ def fit_dki(data_files, bval_files, bvec_files, mask=None,
> >      names = ['FA', 'MD', 'AD', 'RD', 'MK', 'AK', 'RK', 'params']
> >
> >      if out_dir is None:
> > -        out_dir = op.join(op.split(data_files[0])[0], 'dki')
> > +        out_dir = op.join(op.split(data_files)[0], 'dki')
>
> Nice, right. Then, I'm not sure why this code change would work...
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/yeatmanlab/pyAFQ/pull/21/files/78fe87de46aa2ce9e4a85ba589ff3ab0ff0c874a#r62447142>
>